src/app/config/StakedAssets.tsx

```diff
@@ -14,6 +15,7 @@ export const assets: StakeAsset[] = [
   {
     assetName: "Native ETH",
```

Comment:
Should remove eth and eth lsts until we actually integrate with eigen.

cc @kiasaki

Reply:
In the process of moving to .env files and will disable until we're ready for it.